lib/membrane_rtc_engine/engine.ex

```diff
@@ -1083,6 +1101,10 @@ defmodule Membrane.RTC.Engine do
     |> Map.get(track_id)
   end
 
+  defp endpoint_ref(endpoint_id) do
+    Membrane.Child.ref(:endpoint, group: endpoint_id)
+  end
+
```

Comment:
this function is not worth having IMO, especially that `Membrane.Child` is aliased